### ISSUE TITLE
Error running example classifier assertion

### ISSUE THREAD

**Issue by lacom** [promptfoo/promptfoo#1082]
Version 0.68.2

Take the example config generated by `init` and add a classifier type assertion to the test that uses a HuggingFace model. This classifier example is taken straight from the docs:

```
# Learn more about building a configuration: https://promptfoo.dev/docs/configuration/guide
description: "My eval"

prompts:
  - "Write a tweet about {{topic}}"
  - "Write a concise, funny tweet about {{topic}}"
  

providers:
  - "openai:gpt-3.5-turbo"
  - "openai:gpt-4o"
  

tests:
  - vars:
      topic: new york city
    assert:
      # For more information on model-graded evals, see https://promptfoo.dev/docs/configuration/expected-outputs/model-graded
      - type: llm-rubric
        value: ensure that the output is funny

      - type: classifier
        provider: huggingface:text-classification:facebook/roberta-hate-speech-dynabench-r4-target
        value: nothate
        threshold: 0.5
```

Running `eval` fails with error:
`TypeError: Cannot assign to read only property 'options' of object '#<Object>' at runAssertion...`

**Comment by typpo** [comment]
Thanks for flagging!  #1083 should fix this

**Comment by mldangelo** [comment]
@lacom Thanks for reporting this one too! A fix was also deployed in [0.68.3](https://github.com/promptfoo/promptfoo/releases/tag/0.68.3) 

Please let us know if you run into any other issues!

**Comment by lacom** [comment]
@mldangelo excellent, appreciate you fixing!
